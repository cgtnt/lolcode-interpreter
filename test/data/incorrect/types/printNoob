HAI

HOW IZ I ADD YR A AN YR B
  SUM OF A AN B
  GTFO
IF U SAY SO

VISIBLE I IZ ADD YR 2 AN YR 3 MKAY!

KTHXBYE
