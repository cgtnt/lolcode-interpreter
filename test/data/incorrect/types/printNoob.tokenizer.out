@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_FUNC_BEGIN:HOW IZ I-@3:T_T_IDENTIFIER:ADD-@3:T_ARG:YR-@3:T_T_IDENTIFIER:A-@3:T_AND:AN-@3:T_ARG:YR-@3:T_T_IDENTIFIER:B-@4:T_COMMAND_TERMINATOR:
-@4:T_PLUS:SUM OF-@4:T_T_IDENTIFIER:A-@4:T_AND:AN-@4:T_T_IDENTIFIER:B-@5:T_COMMAND_TERMINATOR:
-@5:T_RETURN_NULL:GTFO-@6:T_COMMAND_TERMINATOR:
-@6:T_FUNC_END:IF U SAY SO-@8:T_COMMAND_TERMINATOR:
-@8:T_WRITE_STDOUT:VISIBLE-@8:T_FUNC_CALL:I IZ-@8:T_T_IDENTIFIER:ADD-@8:T_ARG:YR-@8:T_T_INT:2-@8:T_AND:AN-@8:T_ARG:YR-@8:T_T_INT:3-@8:T_END_INF:MKAY-@8:T_BANG:!-@10:T_COMMAND_TERMINATOR:
-@10:T_END:KTHXBYE-@11:T_COMMAND_TERMINATOR:
-@11:T_EOF:
